README.md

```diff
@@ -1,23 +1,48 @@
 # Kubernetes Cluster API Azure Provider  [![Go Report Card](https://goreportcard.com/badge/sigs.k8s.io/cluster-api-provider-azure)](https://goreportcard.com/report/kubernetes-sigs/cluster-api-provider-azure)
 
 ## Getting Started
-### Prerequisites
-1. Install [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/).
-2. Install [minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/) and a [minikube driver](https://github.com/kubernetes/minikube/blob/master/docs/drivers.md). It is recommended to use KVM2 driver for Linux and VirtualBox for MacOS.
-3. Install [kustomize](https://github.com/kubernetes-sigs/kustomize/blob/master/docs/INSTALL.md).
-4. Clone the Project.
-    ```bash
-    git clone https://github.com/kubernetes-sigs/cluster-api-provider-azure $(go env GOPATH)/src/sigs.k8s.io/cluster-api-provider-azure
-    ```
-5. Ensure that you have the project root as your current working directory.
-   ```bash
-   cd $(go env GOPATH)/src/sigs.k8s.io/cluster-api-provider-azure
-   ```
-6. Build the `clusterctl` tool.
 
-   ```bash
-   make clusterctl
-   ```
+### Requirements
+
+- Linux or MacOS (Windows isn't supported at the moment)
+- A set of Azure credentials sufficient to bootstrap the cluster (an Azure service principal with Collaborator rights).
+- [KIND](https://sigs.k8s.io/kind)
```

Comment:
In the interest of consistency, let's move the url below the list and add the url reference here instead.